protocol/app/upgrades/v3.0.0/upgrade.go

```diff
@@ -1,169 +0,0 @@
-package v_3_0_0
```

Comment:
Can we keep this old upgrade handler around? If I understand correctly it doesn't cost anything.

 If we want to cherry-pick something that needs to modify the upgrade handler, it will be nice to have still.

Reply:
I can keep but I suspect the merge pain of developing here and then porting it to 0.47 would be annoying when cherry-picking on to 3.0.0 release branch.

Reply:
Oh good point. Then I don't care either way.